### PR TITLE
[Seq] Give `CompRegOp` an optional name

### PR DESCRIPTION
**Pull request by teqdruid**

### CODE REVIEW

File: lib/Dialect/Seq/SeqPasses.cpp [hunk 1]
```diff
@@ -46,9 +46,6 @@ struct CompRegLower : public OpConversionPattern<CompRegOp> {
     DictionaryAttr regAttrs = reg->getAttrDictionary();
     if (!regAttrs.empty())
       svReg->setAttrs(regAttrs);
```
Comment (lattner) [inline lib/Dialect/Seq/SeqPasses.cpp:48]: Please manually set the attributes in question, e.g. by building the op with its builder.  Please don't "just pass on" whatever attributes are on the seq register.

**Review by lattner — APPROVED** [review]
Thanks for tackling this!

**Comment by teqdruid** [comment]
Canceled build but GH is not letting me restart it. Merging since it passes tests locally.

**Comment by teqdruid** [comment]
Thanks for the review @lattner !

**Pull request merged**